services/slam/fake/slam.go

```diff
@@ -105,5 +106,6 @@ func (slamSvc *SLAM) GetLimits(ctx context.Context) ([]referenceframe.Limit, err
 	return []referenceframe.Limit{
 		{Min: dims.MinX, Max: dims.MaxX},
 		{Min: dims.MinY, Max: dims.MaxY},
+		{Min: -2 * math.Pi, Max: 2 * math.Pi},
```

Comment:
[minor] We shouldn't be modifying this function here, since SLAM is only ever concerned with 2DOF